examples/shaders/resources/shaders/glsl430/golRender.glsl

```diff
@@ -0,0 +1,24 @@
+// Game of Life rendering shader
+// Just renders the content of the ssbo at binding 1 to screen.
+#version 430

```

Comment:
Could you rename the shaders to follow `snake_case` convention, raylib uses it for external files/resources. 
In this case it should be `gol_render.glsl`